package.json

```diff
@@ -22,7 +22,9 @@
     "svg:win": "if not exist dist md dist && yarn svg",
     "svg": "svg-sprite-generate -d src/assets/ -o dist/sprite.svg",
     "pull_tools": "git submodule update --init --recursive",
-    "checkout_tools": "git submodule foreach git pull origin master"
+    "checkout_tools": "git submodule foreach git pull origin master",
+    "test:cypress": "cypress open",
+    "test:dev": "node ./test_server/express.js"
```

Comment:
this step should be included to the `test:e2e` command

Reply:
done

Reply:
I tried that but it didnt work. We'll discuss about this. I have also added nodemon instead of normal node command for faster development.